google-cloud-spanner/src/main/java/com/google/cloud/spanner/InstanceAdminClientImpl.java

```diff
@@ -47,6 +47,8 @@ protected Policy fromPb(com.google.iam.v1.Policy policyPb) {
     }
   }
 
+  static final String NOT_BOTH_NODE_COUNT_AND_PROCESSING_UNITS =
```

Comment:
This field is superfluous and adds an unnecessary indirection. Inline it. In general, do not use named constants for user visible strings.

Reply:
Inlined it.